util/migration/src/lib.rs

```diff
@@ -130,12 +130,14 @@ pub trait Migration: 'static {
 	fn migrate(&mut self, source: Arc<Database>, config: &Config, destination: &mut Database, col: Option<u32>) -> Result<()>;
 }
 
-/// A simple migration over key-value pairs.
+/// A simple migration over key-value pairs of a single column.
 pub trait SimpleMigration: 'static {
 	/// Number of columns in database after the migration.
 	fn columns(&self) -> Option<u32>;
 	/// Version of database after the migration.
 	fn version(&self) -> u32;
+	/// Index of column which should be migrated.
+	fn migrated_column_index(&self) -> Option<u32>;
```

Comment:
without this function, the trait is completely useless, cause we don't know which column we are currently migrating